API.md

```diff
@@ -89,7 +89,7 @@ The environment's cluster must not be imported.
 
 ---
 
-##### `isConstruct` <a name="isConstruct" id="@aws-cdk-containers/ecs-service-extensions.Environment.isConstruct"></a>
```

Comment:
Why are you making changes to this file as part of moving `aws-cdk-lib` and `constructs` to peer dependencies? Shouldn't that change have no impact here?

Reply:
This came from one of the generation projen commands during build. I believe it changed because the running version of aws-cdk-lib in this repo was actually lowered.

Reply:
Okay, thank you 👍